provider/README.md

```diff
@@ -0,0 +1,14 @@
+# Unlock RPC provider
+
+This provider is a proxy application that connects to all networks supported by Unlock.
+TODO: only support RPC calls to Unlock contracts (or related contracts... such as ERC20 contracts).
+
+Even though this adds centralization (and effectively introduces a middleman), this is solely provided as a developer tool for convenience. Your application should use its own RPC endpoint in production.
+
```

Comment:
It will good to add an example of what a RPC call looks like.  
I guess it's sth like `https://rpc.unlock-protocol.com/<chainId>/<call logic>` ?

Reply:
or maybe a link to the doc can do. too

Reply:
Oh they are _exactly_ like other ethereum JSON RPC calls... I'd rather not document them here (because they change and I don't want to maintain the list here) but I'll add a link.